snap/info.go

```diff
@@ -120,6 +120,7 @@ type SideInfo struct {
 	SnapID            string   `yaml:"snap-id" json:"snap-id"`
 	Revision          Revision `yaml:"revision" json:"revision"`
 	Channel           string   `yaml:"channel,omitempty" json:"channel,omitempty"`
+	Contact           string   `yaml:"contact,omitempty" json:"contact,omitempty"`
```

Comment:
total personal nitpick, but I don't think the order is fully lexical here anyway, I would have put this before Private (like in seed)